integration-tests/helpers/embrace_api.ts

```diff
@@ -0,0 +1,46 @@
+import {Data, EmbraceData} from "../typings/embrace";
+import axios from "axios";
+import {RAW_EMB_URL, SERVICE_NAME} from "./env";
+const EMB_URL = RAW_EMB_URL.replace("{servicename}", SERVICE_NAME);
+type SESSION_ID = string | string[];
+
+const getSessionPayloads = async (
+  sessionId: SESSION_ID,
+  delay = 4000,
+): Promise<EmbraceData> => {
+  if (delay) {
+    console.log(`waiting ${delay}ms before checking for span payloads`);
+    await new Promise(r => setTimeout(r, delay));
+  }
+  const response = await axios.get(EMB_URL);
+
+  console.log("SESSION ID: ", sessionId);
+
+  const responseData = (await response.data) as EmbraceData;
+
+  const getDataFromSessionId = (array = [] as Data[], sessionId: SESSION_ID) =>
+    array.filter(r => {
+      if (typeof sessionId === "string") {
+        if (r.Body.includes(sessionId)) {
+          return true;
+        }
+      } else if (sessionId instanceof Array) {
+        return sessionId.some(sId => r.Body.includes(sId));
+      }
+    });
+
+  const dataFiltered = {
+    Spans: [],
+    Logs: [],
+  };
+
+  Object.entries(responseData).forEach(([key, value]) => {
+    dataFiltered[key] = getDataFromSessionId(value, sessionId).map(item =>
+      JSON.parse(item.Body),
```

Comment:
for spans should parse this using `parseSpanPayload` that's used in `helpers/embrace_server.ts`. Actually should get rid of `helpers/embrace_server.ts` altogether or have this replace it, ideally the specs that are calling it would behave the same if they get back the same ParsedSpanPayload type that was being used before from the local mocked server